Apps/Admin/Client/Components/MessageVerificationTable.razor

```diff
@@ -40,12 +40,10 @@
     <RowTemplate>
         <MudTd data-testid="messaging-verification-table-email-sms" DataLabel="Email/SMS">@context.EmailOrSms</MudTd>
         <MudTd data-testid="messaging-verification-verified" DataLabel="Verified">@context.Verified</MudTd>
-        <MudTd data-testid="messaging-verification-table-date" DataLabel="Verification Date">@DateFormatter.ToShortDateAndTime(@context.VerificationDate)</MudTd>
+        <MudTd data-testid="messaging-verification-table-date" DataLabel="Verification Date">@DateFormatter.ToShortDateAndTime(context.VerificationDate.ToLocalTime())</MudTd>
```

Comment:
Using ToLocalTime() is not going to work.  There is no guarantee, the server the code will be run is Pacific time.

The conversion to PST from UTC should be done in an auto mapper util via SupportService.GetMessageVerifications.  See AdminUserProfileMapUtils.